src/index.css

```diff
@@ -1,5 +1,12 @@
 @import url('https://fonts.googleapis.com/css?family=Inter&display=swap');
 
+html,
+body,
+#root,
+.App {
+  height: 100%;
+}
+
 html {
   height: 100%;
 }
```

Comment:
```suggestion
```

You could remove `height` across html, body, and etc, right?

Reply:
removing this causes the height to only be the height of the content:

![Screen Shot 2020-03-31 at 10 59 33 AM](https://user-images.githubusercontent.com/19784034/78059474-c0523100-733e-11ea-89aa-bf878249bce2.png)

Reply:
But I thought this rule would cover it:

```html,
 body,
 #root,
 .App {
   height: 100%;
 }```

Reply:
oh yeah you're right. removing!